File: Assignments/2_Operators and Expression/Javascript/Solutions/Q6.js
function calculateDistance(u, t, a) {
    return u * t + 0.5 * a * t * t;
}

const u = 5; // initial velocity
const t = 10; // time
const a = 2; // acceleration
const distance = calculateDistance(u, t, a);
console.log(`Distance traveled: ${distance}`);

File: Assignments/2_Operators and Expression/C#/Q6.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter initial velocity (u): ");
        double u = double.Parse(Console.ReadLine());
        Console.Write("Enter time (t): ");
        double t = double.Parse(Console.ReadLine());
        Console.Write("Enter acceleration (a): ");
        double a = double.Parse(Console.ReadLine());

        double s = u * t + 0.5 * a * t * t;
        Console.WriteLine("Distance traveled: " + s);
    }
}
